backend/src/test/java/com/soen341/instagram/dao/impl/PictureRepositoryTest.java

```diff
@@ -0,0 +1,50 @@
+package com.soen341.instagram.dao.impl;
+
+import com.soen341.instagram.model.Account;
+import com.soen341.instagram.model.Picture;
+import org.junit.jupiter.api.BeforeEach;
```

Comment:
Nitpicky comment: same as the other import comment (I just think it'd be nicer to distinguish them but that might just be me 😅)